static/styles/lib/header.scss

```diff
@@ -39,8 +39,17 @@
 }
 
 .navbar-nav {
+
+	@include media-breakpoint-down(md) {
+		height: 56px;
+  }
+
+	@include media-breakpoint-down(sm) {
+		height: 56px;
+  }
 	.minimal-button {
 		float: right;
+		margin-top: 3px;
```

Comment:
same here, as the more specificated values follow rem values, I would not like to have different units as base